src/App.css

```diff
@@ -12,28 +13,48 @@ nav {
 
 nav > * {
```

Comment:
- [ ] You have a great navbar but it seems that you forgot the `user` icon on the top right corner also your books and category links do not have the text-decoration please check the original design

Your design 👇 
![Screen Shot 2022-06-10 at 20 49 54](https://user-images.githubusercontent.com/75126481/173131299-4d703de5-e57c-4a9c-b700-412e6ae219f1.png)

The original Design 👇 

![Screen Shot 2022-06-10 at 21 05 31](https://user-images.githubusercontent.com/75126481/173133463-c9e7ba9d-306f-4403-b819-78c45abb4e00.png)

Reply:
@hiromataba I added the user but it seems you already approve it. You can take a look if you like. Thanks! 😉

Reply:
@marurevi i can see it yeah, Great design 😃